src/strict/api/routes.py

```
@@ -115,3 +129,71 @@ async def process_request(
+                # Stream results back
+                async for chunk in processor.stream_process(request):
+                    await websocket.send_json({"type": "chunk", "content": chunk})
+
+                await websocket.send_json({"type": "done", "content": ""})
+
+            except ValidationError as e:
+                await websocket.send_json({"type": "error", "content": str(e)})
+            except Exception as e:
+                logger.exception("Error processing WebSocket request")
+                await websocket.send_json(
+                    {"type": "error", "content": f"Internal error: {str(e)}"}
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)